lua/wikis/commons/Widget/MainPage/ThisDay/Content.lua

```diff
@@ -8,37 +8,58 @@
 local Lua = require('Module:Lua')
 
 local Class = Lua.import('Module:Class')
+local DateExt = Lua.import('Module:Date/Ext')
+local Info = Lua.import('Module:Info', {loadData = true})
+local Logic = Lua.import('Module:Logic')
 local Page = Lua.import('Module:Page')
 local String = Lua.import('Module:StringUtils')
-local Template = Lua.import('Module:Template')
 
 local Widget = Lua.import('Module:Widget')
 local HtmlWidgets = Lua.import('Module:Widget/Html/All')
 local Div = HtmlWidgets.Div
 local Link = Lua.import('Module:Widget/Basic/Link')
 local Small = HtmlWidgets.Small
+local ThisDayBirthday = Lua.import('Module:Widget/ThisDay/Birthday')
+local ThisDayPatch = Lua.import('Module:Widget/ThisDay/Patch')
+local ThisDayTournament = Lua.import('Module:Widget/ThisDay/Tournament')
 local WidgetUtil = Lua.import('Module:Widget/Util')
 
+---@type ThisDayConfig
+local Config = Info.config.thisDay or {}
+
 ---@class ThisDayContent: Widget
----@field props { month: integer?, day: integer?, birthdayListPage: string? }
+---@field props { month: integer?, day: integer?, birthdayListPage: string?, noTwitter: boolean?}
 ---@operator call(table): ThisDayContent
 local ThisDayContent = Class.new(Widget)
 ThisDayContent.defaultProps = {
-	month = tonumber(os.date('%m')),
+	month = DateExt.getMonthOf(),
 	day = tonumber(os.date('%d'))
 }
 
 function ThisDayContent:render()
 	local month = self.props.month
 	local day = self.props.day
-	local frame = mw.getCurrentFrame()
 	local birthdayListPage = self.props.birthdayListPage
 	local showBirthdayList = String.isNotEmpty(birthdayListPage) and Page.exists(birthdayListPage --[[@as string]])
 	return WidgetUtil.collect(
 		Div{
 			attributes = { id = 'this-day-facts' },
 			children = {
-				Template.safeExpand(frame, 'Liquipedia:This day/' .. month .. '/' .. day)
```

Comment:
This will remove the ability for trivia? (not that there's much if any?). 

If we remove the trivia, then the "add trivia" link should be removed (L77-86)

Reply:
imo fine to remove entirely

Reply:
@Kanopedia

Reply:
I think even though it isn't used as much, I'd still prefer to keep this. (Atleast I've had some occasions on LoL where there actually was some interesting things that popped up) But I can also bring this forward in the community team meeting later today